apps/nowcasting-app/.gitignore

```diff
@@ -36,5 +36,3 @@ yarn-error.log*
 # Sentry
 .sentryclirc
 
-# vscode 
-/.vscode/
```

Comment:
should you keep this ignored?

Reply:
that should be fine, because i deleted the .vscode file that appeared when i was doing linting stuff the other week. feels like it's ok to not ignore a file that doesn't exist anymore, right?